### PR TITLE
Node dependency updates

### PR DESCRIPTION
**Pull request by hatched**
Both ESLint and Babel had major version updates so this updates those along with all the other updated dependencies.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4479/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4480/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4481/
Test PASSed.

**Comment by hatched** [comment]
#### To QA

```
npm cache clear
make clean-all deps fast-babel run
```
Go all chaos-monkey on the GUI to make sure it works as expected.

File: jujugui/static/gui/src/app/components/entity-content/entity-content.js [hunk 1]
```diff
@@ -143,11 +143,11 @@ YUI.add('entity-content', function() {
     */
     _generateDescription: function(entityModel) {
       if (entityModel.get('entityType') === 'charm') {
-        var bugLink = `https://bugs.launchpad.net/charms/+source/` +
+        var bugLink = 'https://bugs.launchpad.net/charms/+source/' +
           `${entityModel.get('name')}`;
-        var submitLink = `https://bugs.launchpad.net/charms/+source/` +
+        var submitLink = 'https://bugs.launchpad.net/charms/+source/' +
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/entity-content/entity-content.js]: This is an interesting thing to pick up, I guess it makes sense!

**Comment by huwshimi** [comment]
Looks great @hatched QA OK :+1:!

**Comment by frankban** [comment]
:+1: :shipit: 
Thank you!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by hatched** [comment]
Thanks! :shipit:

**Pull request merged**